distributed/protocol/tests/test_protocol.py

```diff
@@ -251,6 +254,22 @@ def test_large_bytes():
         assert len(frames[1]) < 1000
 
 
+@pytest.mark.parametrize(
+    "Wrap,Wrapped",
+    [
+        (Serialize, Serialized),
+        (to_serialize, Serialized),
+        (ToPickle, Pickled),
+    ],
+)
+def test_sizeof_serialize(Wrap, Wrapped):
+    size = 100_000
+    ser_obj = Wrap(b"0" * size)
+    assert size <= sizeof(ser_obj) < size * 1.05
```

Comment:
allowing 5% overhead is a bit generous but this test doesn't really care if the wrapper would add more metadata to it or if python object sizes changed, etc.